README.md

```diff
@@ -1,79 +1,128 @@
-* [Accord Project Blog][apblog]
-* [Accord Project Slack][apslack]
-* [Accord Project Technical Documentation][apdoc]
-* [Accord Project GitHub][apgit]
-
-
-### Documentation
-* [Getting Started with Accord Project][docwelcome]
-* [Concepts and High-level Architecture][dochighlevel]
-* [How to use the Cicero Templating System][doccicero]
-* [How to Author Accord Project Templates][docstudio]
-* [Ergo Language Guide][docergo]
-
-## Contributing
-
-The Accord Project technology is being developed as open source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
-
-Find out what’s coming on our [blog][apblog].
-
-Join the Accord Project Technology Working Group [Slack channel][apslack] to get involved!
-
-For code contributions, read our [CONTRIBUTING guide][contributing] and information for [DEVELOPERS][developers].
-
-## License <a name="license"></a>
-
-Accord Project source code files are made available under the [Apache License, Version 2.0][apache].
-Accord Project documentation files are made available under the [Creative Commons Attribution 4.0 International License][creativecommons] (CC-BY-4.0).
-
-Copyright 2018-2019 Clause, Inc. All trademarks are the property of their respective owners. See [LF Projects Trademark Policy](https://lfprojects.org/policies/trademark-policy/).
-
-[linuxfound]: https://www.linuxfoundation.org
-[charter]: https://github.com/accordproject/techdocs/blob/master/CHARTER.md
-[apmain]: https://accordproject.org/ 
-[apworkgroup]: https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MjZvYzIzZHVrYnI1aDVzbjZnMHJqYmtwaGlfMjAxNzExMTVUMjEwMDAwWiBkYW5AY2xhdXNlLmlv&tmsrc=dan%40clause.io
-[apblog]: https://medium.com/@accordhq
-[apnews]: https://www.accordproject.org/news/
-[apgit]:  https://github.com/accordproject/
-[apdoc]: https://docs.accordproject.org/
-[apslack]: https://accord-project-slack-signup.herokuapp.com
-
-[docspec]: https://docs.accordproject.org/docs/spec-overview.html
-[docwelcome]: https://docs.accordproject.org/docs/accordproject.html
-[dochighlevel]: https://docs.accordproject.org/docs/spec-concepts.html
-[docergo]: https://docs.accordproject.org/docs/logic-ergo.html
-[docstart]: https://docs.accordproject.org/docs/accordproject.html
-[doccicero]: https://docs.accordproject.org/docs/basic-use.html
-[docstudio]: https://docs.accordproject.org/docs/advanced-latedelivery.html
-
-[contributing]: https://github.com/accordproject/techdocs/blob/master/CONTRIBUTING.md
-[developers]: https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md
-
-[apache]: https://github.com/accordproject/template-studio-v2/blob/master/LICENSE
-[creativecommons]: http://creativecommons.org/licenses/by/4.0/
+﻿<h1 align="center">
+[![All Contributors](https://img.shields.io/badge/all_contributors-24-orange.svg?style=flat-square)](#contributors-)
```

Comment:
This doesn't render correctly, 
![image](https://user-images.githubusercontent.com/7544022/66821870-1838df00-ef3b-11e9-9f75-f551e64c60ee.png)

Badges should sit below the AP logo

Reply:
the way the library detects the badge zone (in order to update it) is through a regex `var badgeRegex = /\[!\[All Contributors\]\([a-zA-Z0-9\-./_:?=]+\)\]\(#[\w-]+\)/;`, BUT markdown doesn't mix well with html.
I'll keep the logo centered, but I will need to move the badges on the left. Is that acceptable?

Reply:
it will only drive the rewriting of the badge to nothing ^^.
The library has hardcoded the regex to be the markdown tag. You can just change what's it's replaced with.

In order to "fix" this, we would need to PR to the allcontributor, to expose the regex in the options.